src/github.com/matrix-org/dendrite/syncapi/consumers/roomserver.go

```diff
@@ -28,15 +28,15 @@ import (
 	sarama "gopkg.in/Shopify/sarama.v1"
 )
 
-// Server contains all the logic for running a sync server
-type Server struct {
+// Roomserver consumes events that originated in the room server.
+type Roomserver struct {
```

Comment:
The package is already `consumers` so if the sync server uses the type it'll be `consumers.Roomserver` which is nice? Linter has the weird stuttering warning thing, so I figured this would be good.